docs/guide/introduction.rst

```diff
@@ -66,7 +66,11 @@ The commands used for set up the CLI are:
 * the :ref:`cli_plan-rbfe-network`
 * the :ref:`cli_plan-rhfe-network`
 
-.. TODO add command example here;
+For example, you can create an RBFE network using
```

Comment:
RBFE -> Relative Binding Free Energy (RBFE) first use of a the acronym in this document right?